python/lsst/pipe/tasks/mocks/mockObservation.py

```diff
@@ -100,7 +100,16 @@ def run(self, butler, n, tractInfo, camera, catalog=None):
             if not catalog.getSchema().contains(self.schema):
                 raise ValueError("Catalog schema does not match Task schema")
         visit = 1
+
         for position, pa in self.makePointings(n, tractInfo):
+            visitInfo = lsst.afw.image.makeVisitInfo(
+                exposureTime = self.config.expTime,
+                date = lsst.daf.base.DateTime.now(),
+                boresightRaDec = position,
+                # once we know what pa is, we can do something like the following
```

Comment:
It not clear to me what this comment means.

I don't like leaving code in but commented-out, even if there's an explanation.